### ISSUE TITLE
[Bug]: Missing accessibility lable 

### ISSUE THREAD

**Issue by TStArK1233** [Pool-Of-Tears/GreenStash#36]
### Steps to reproduce

![IMG_20230517_202207](https://github.com/Pool-Of-Tears/GreenStash/assets/133884373/5fba3cbf-e275-4d36-a751-24dc9ebea686)

The following buttons marked in red donot have lable so please consider adding lable. Thank You for considering it😊😊.

### Expected behaviour

When talkback is enabled we expect talkback to say edit goal, information and delete goal.

### Actual behaviour

When talkback is on it only says button.

### Device name

Redmi note 9

### Android version

Android 10

### GreenStash version

2.5.0

**Comment by starry-shivam** [comment]
Well tbh this is not really a bug but more like a usability enhancement, hence please recreate this issue as a feature request instead. Thanks!
